moyo.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style2.css">
+    <link rel="stylesheet" href="https://fonts.gstatic.com/s/roboto/v29/KFOmCnqEu92Fr1Mu4mxK.woff2">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+         <ul class="container" >
+            <a href="https://www.moyo.ua/">
+              <img
+                class="logo"
+                src="https://raw.githubusercontent.com/mate-academy/layout_moyo-header/master/src/images/logo.png"
+            ></a>
+            <li class="box box_1">
+              <a id="link" class="is-active" href="https://www.moyo.ua/">Apple</a>
+            </li>
+            <li class="box box_2">
+              <a href="https://www.moyo.ua/" >Samsung</a>
+            </li>
+            <li class="box box_3">
+              <a href="https://www.moyo.ua/">Смартфоны и телефоны</a>
+            </li>
+            <li class="box box_4" data-qa="hover">
+              <a href="https://www.moyo.ua/">Ноутбуки и компьютеры</a>
+            </li>
+            <li class="box box_5">
+              <a href="https://www.moyo.ua/">Гаджеты</a>
+            </li>
+            <li class="box box_6">
+              <a href="https://www.moyo.ua/">Планшеты</a>
+            </li>
+            <li class="box box_7">
```

Comment:
```suggestion
            <li class="box">
              <a href="https://www.moyo.ua/">Планшеты</a>
            </li>
            <li class="box">
```
Для чого ці всі класи із цифрами?